src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt

```diff
@@ -193,20 +198,21 @@ object H2DatabaseEventStore {
     @PublishedApi internal inline fun <reified M: EventMetadata> create(
         synchronousEventProcessors: List<EventProcessor<M>>,
         db: Database,
-        objectMapper: ObjectMapper
+        objectMapper: ObjectMapper,
+        tableName: String
     ): RelationalDatabaseEventStore<M> {
-        return RelationalDatabaseEventStore(db, eventsTable(), synchronousEventProcessors, M::class.java, objectMapper)
+        return RelationalDatabaseEventStore(db, eventsTable(tableName), synchronousEventProcessors, M::class.java, objectMapper)
     }
 
-    @PublishedApi internal fun eventsTable() = Events { name -> this.text(name) }
+    @PublishedApi internal fun eventsTable(tableName: String) = Events(tableName) { name -> this.text(name) }
 }
 
 private fun <T> String.asClass(): Class<out T>? {
     @Suppress("UNCHECKED_CAST")
     return Class.forName(this) as Class<out T>?
 }
 
-class Events(jsonb: Table.(String) -> Column<String>) : Table() {
+class Events(tableName: String, jsonb: Table.(String) -> Column<String>) : Table(tableName) {
```

Comment:
Small thing. Every other reference to tableName comes after the objectMapper.

Reply:
Hmm, one reason it's nice to put this argument first is that it's going up to the parent.

Reply:
Decided to leave this in first position for the above reason - I think it's ok since `jsonb` isn't `objectMapper` so it's not inconsistent, although I can see it kinda feels that way :)